cvat-ui/src/components/annotation-page/review/issue-dialog.tsx

```diff
@@ -66,6 +73,38 @@ export default function IssueDialog(props: Props): JSX.Element {
         }
     }, [resolved]);
 
+    useEffect(() => {
+        if (ref.current && clickCoordinates && canvasRect) {
```

Comment:
let's add a small comment why we need this effect

Reply:
Should not we move this logic to issue aggregator? (to cover dialogs for existing issues and new issues by the same code)